src/sql/src/func.rs

```diff
@@ -2339,6 +2343,13 @@ pub static PG_CATALOG_BUILTINS: Lazy<BTreeMap<&'static str, Func>> = Lazy::new(|
         "pg_postmaster_start_time" => Scalar {
             params!() => UnmaterializableFunc::PgPostmasterStartTime => TimestampTz, 2560;
         },
+        "pg_relation_size" => Scalar {
+            params!(RegClass, String) => sql_impl_func("0::pg_catalog.int8") => Int64, 2332;
+            params!(RegClass) => sql_impl_func("0::pg_catalog.int8") => Int64, 2325;
+        },
+        "pg_stat_get_numscans" => Scalar {
+            params!(Oid) => sql_impl_func("0::pg_catalog.int8") => Int64, 1928;
+        },
```

Comment:
I think for these functions, we'd be better off returning -1 rather than 0. 0 is a valid but (most of the time) incorrect answer and may be misinterpreted as accurate. While -1 is totally invalid and can't be misinterpreted.

Reply:
Unless it causes errors in whatever is consuming this function. In that case we're probably fine to keep it at 0.

Reply:
tbh, I only tried with 0. Happy to try to -1

Reply:
-1 works!